cypress/fixtures/globalSelectors.json

```diff
@@ -0,0 +1,5 @@
+{
+    "successMessage": ".message-success.success",
+    "errorMessage": ".message-error",
```

Comment:
Not sure how much these selectors differ between hyvä and luma? I've used the commands in luma only so far, so I can't say a lot about hyvä.